src/app/components/services/metadata.service.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Created by bo on 5/23/16.
+ */
+'use strict';
+
+angular.module('transmartBaseUi').factory('isURL', function () {
+
+  return function (s) {
+    var regexp = /(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?/
+    return regexp.test(s);
+  }
+
+});
+
```

Comment:
this file shouldn't be here right? because it's already replaced by util.service?